### PR TITLE
Added p1 and p2 variants

### PR DESCRIPTION
**Pull request by marques-valeria**

### CODE REVIEW

**Comment by ayakayorihiro** [comment]
These changes look good to me; will also check the PR on STARTS.

File: emop-maven-plugin/src/main/java/edu/cornell/ImpactedClassMojo.java [hunk 1]
```diff
@@ -15,9 +16,22 @@ public class ImpactedClassMojo extends ImpactedMojo {
     @Parameter(property = "updateChecksums", defaultValue = "true")
     private boolean updateChecksums;
 
+    /**
+     * Parameter to determine which closure to use for impacted classes.
+     * TRANS_OF_INVERSE_TRANS = p1
+     * TRANS_AND_INVERSE_TRANS = p2
+     * TRANS = p3
```
Comment (owolabileg) [inline emop-maven-plugin/src/main/java/edu/cornell/ImpactedClassMojo.java:23]: TRANS --> TRANSITIVE

File: emop-maven-plugin/src/main/java/edu/cornell/ImpactedClassMojo.java [hunk 2]
```diff
@@ -15,9 +16,22 @@ public class ImpactedClassMojo extends ImpactedMojo {
...
+     */
+    @Parameter(
+            property = "closureOption",
+            defaultValue = "TRANS"
```
Comment (owolabileg) [inline emop-maven-plugin/src/main/java/edu/cornell/ImpactedClassMojo.java:27]: TRANS --> TRANSITIVE

File: emop-maven-plugin/src/main/java/edu/cornell/ImpactedClassMojo.java [hunk 3]
```diff
@@ -15,9 +16,22 @@ public class ImpactedClassMojo extends ImpactedMojo {
...
+    )
+    private TransitiveClosureOptions closureOption;
+
     public void execute() throws MojoExecutionException {
         setUpdateImpactedChecksums(updateChecksums);
         setTrackNewClasses(true);
+        setTransitiveClosureOption(closureOption);
```
Comment (owolabileg) [inline emop-maven-plugin/src/main/java/edu/cornell/ImpactedClassMojo.java:34]: I am assuming that his method is in STARTS, and that is why its implementation is not in this diff?

**Review by owolabileg — CHANGES_REQUESTED** [review]

File: emop-maven-plugin/src/main/java/edu/cornell/ImpactedClassMojo.java (hunk 3 above)
Comment (marques-valeria) [reply emop-maven-plugin/src/main/java/edu/cornell/ImpactedClassMojo.java:34]: Yes this method is with the other setters in STARTS

**Comment by owolabileg** [comment]
I merged the dependent PR in STARTS.

Please let let me know when this one is ready to merge.

**Pull request merged**